src/Controller/HomeController.php

```diff
@@ -12,8 +12,41 @@ class HomeController extends AbstractController
      */
     public function index()
     {
-        return $this->render('home/index.html.twig', [
-            'someVariable' => 'NFQ Akademija',
-        ]);
+
+        $group = $this->groupByStudents();
+        $projects = $this->getProjects();
+        return $this->render('home/index.html.twig', ['group' => $group, 'projects' => $projects]);
+    }
+
+    protected function getProjects()
+    {
+        $projects = $this->getProjectData();
+        $result = [];
+        foreach ($projects as $projectName => $project) {
+            if (!in_array($projectName, $result)) {
+                $result[] = ['project' => $projectName, 'name' => $project['name']];
+            }
+        }
+        return $result;
+    }
+
+
+    private function getProjectData()
+    {
+        $projectFile = file_get_contents('../public/students.json');
```

Comment:
```suggestion
        $projectFile = file_get_contents(__DIR__ . '/../public/students.json');
```
Dabar veiks. Bet ateityje rašant kelią iki failo, verta nepasitikėti _working directory_.
Alternatyva naudoti `KernelInterface $kernel`

Plačiau:
 * https://www.php.net/manual/en/language.constants.predefined.php
 * https://www.php.net/manual/en/function.set-include-path.php
 * https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/HttpKernel/KernelInterface.php#L137